evap/results/templates/results_evaluation_detail.html

```diff
@@ -26,43 +26,55 @@
                 <span class="fas fa-lock"></span> {% trans 'This evaluation is private. Only contributors and participants can see the results.' %}
             </div>
         {% endif %}
-        <div class="d-flex">
-            <h3 class="me-auto">{{ evaluation.full_name }} ({{ evaluation.course.semester.name }})</h3>
+        <div class="row align-items-end mb-3">
+            <div class="col">
+                <h3 class="mb-0">{{ evaluation.full_name }} ({{ evaluation.course.semester.name }})</h3>
+            </div>
 
-            <div class="btn-switch btn-switch-light my-auto d-print-none">
-                <div class="btn-switch-label">{% trans 'View' %}</div>
-                <div class="btn-switch btn-group">
-                    {% if user.is_staff and view == 'export' or is_contributor %}
-                        <a href="{% if is_contributor %}{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view=export{% endif %}" role="button" class="btn btn-sm btn-light{% if view == 'export' %} active{% endif %}" data-bs-toggle="tooltip" data-bs-placement="bottom"
-                        title="{% blocktrans %}Shows filtered view meant for personal export. Other contributors' results and private answers are hidden.{% endblocktrans %}">
-                        {% trans 'Export' context 'view mode' %}
+            <div class="col-auto">
+                <div class="btn-switch btn-switch-light my-auto d-print-none">
+                    <div class="btn-switch-label">{% trans 'View' %}</div>
+                    <div class="btn-switch btn-group">
+                        {% if user.is_staff and view == 'export' or is_contributor %}
+                            <a href="
+                                    {% if is_contributor %}{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view=export{% endif %}"
+                               role="button" class="btn btn-sm btn-light{% if view == 'export' %} active{% endif %}"
+                               data-bs-toggle="tooltip" data-bs-placement="bottom"
+                               title="{% blocktrans %}Shows filtered view meant for personal export. Other contributors' results and private answers are hidden.{% endblocktrans %}">
+                                {% trans 'Export' context 'view mode' %}
+                            </a>
```

Comment:
(Wait for @niklasmohrin to confirm before doing new formatting)

We don't use a html formatter (yet) because we're unaware of one that handles django's templating language well. Currently, the code base uses either a very packed style (pretty much all attributes on one line, this is what this file used before, with the exception that the long title was put on a separate line), or we use what [Prettier](https://prettier.io/) suggests for HTML with additional manual adjustments for the django template language constructs.

For consistency, I'd like to stick to one of these styles. See [the prettier.io playground for this snippet](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAhgAgBYCc4BmAvADojACkmAlgTQM4D6ksu1ARgK4wS6YUBfSpk64ANpgDk+epzEx6SOADd0YzuhjVojACZwY6amMmYVajVugA6SKPpxrDgLZx68XNeq6zq9Zu0oLx9BAH5lajgAdyI4AA8AB14YYQRdWn4BMhIoTDz8gsKi4vzcCDE4UhAuGB4oMkwwMXR6eir2GFyOqABaemdMbp6xagBzbBSqDIjozCIiKXik3BhTQUx0MC1lOFSodLpBbNyS07P83U10HvZ6Hp5R0YqqnnKtBIbLwxu7hOawOCuWDtCC1CDOY7nKElLQwZ5kYTsMQQMAAaxguHQUHomQAytgIFEcQRjB44D4ZlFMK4sTBMAReJgEnBcPRoGozIlktZMAB5GDYFmNaAYjjcXj0UwyOQKDb7JlsVTwOX0KIsnHofA4bz6IJ7XRIlHozHYzJkAB8OWh1sKwgxWJxkgAolyVqYWPA4nTJJTqRB9GssicbedUAB6dDmkAAGhAEASVmxyFAmrKUQACpqEPRkCA1FF0ABPHOx9iYtEGXHoVwAGWoUDgyAIagcpfLqMrCU29dGyAxnDgsZc1D7uAHQ57FQAipxQY2kM2xK2QAArehxXGTuAzudNluDkAAR1n8HTZQSOZQLR6DfJ5JjIHtxh7AGFwc50Lm1GIH-QtwBBWo2BqOB0xZOsGz3JcDwmZwxAAdWwah4HoLsAVxbNkOoCIYELXMwFaB9lAHABJfYEBgXEwDYBN-32XFcIqKDlwSMoHHgzEElzVi3BZHYH3rBwVjPdBRg-ZiDy7VkWVzWCf1jfBj2ofARLEz8F33WNDHYeDvAFZAABZY04BwABV0FuCTY0BdhyX0XQayxUYNFGOAADFeA-Woey-cUH1Y+sYF03R9KQABGAAmAAGAQBCAA) for the prettier.io formatting.

Also applies to other changes in HTML files

Reply:
Sorry, I wasn't aware that there is no tooling set up to format/validate the HTML/Django templates. (I just relied on my IDEs auto-formatting and assumed the precommit script would complain if it's against the project style)

I reformatted all my changes to use the "prettier" version.

Should I also reformat the rest of the file to fit this style?

Reply:
I think the current state is fine, thanks 👍